src/components/TopBar.js

```diff
@@ -276,14 +275,7 @@ export default Utils.connectAtom(authStore, 'authState')(class TopBar extends Co
         ])
       ]),
       children,
-      navShown && this.buildNav(),
-      showingSupportModal && h(SupportRequestModal, {
-        onDismiss: () => this.setState({ showingSupportModal: false }),
-        onSuccess: () => {
```

Comment:
This doesn't appear to have been ported over to the new implementation and may be causing the error.

Reply:
I'm seeing the same error! I agree with Kate; I think that it's because onSuccess was passed in as a prop, and the submit function calls it.

Since these changes make it so we aren't using state to display the form, it seems onSuccess would now only serve to notify the user that the message was sent, so I think just directly putting the notification in submit() would solve it

Reply:
Thank you both. I'm sorry I didn't do the testing I should have. I'll let you know when I believe it to be fixed.